controllers/osbuildenvconfig_controller.go

```diff
@@ -542,6 +570,37 @@ func (r *OSBuildEnvConfigReconciler) ensureWorkersExists(ctx context.Context, re
 	return false, nil
 }
 
+func (r *OSBuildEnvConfigReconciler) ensureWorkersConfigured(ctx context.Context, reqLogger logr.Logger, instance *osbuildv1alpha1.OSBuildEnvConfig) (bool, error) {
+	for i := range instance.Spec.Workers {
+		configured, err := r.ensureWorkerConfigured(ctx, reqLogger, instance, &instance.Spec.Workers[i])
+		if err != nil {
+			return false, err
```

Comment:
maybe you can use the condition message to update that not all the workers were configured properly.